website/scripts/sync-algolia-crawler.mjs

```
@@ -0,0 +1,95 @@
+const authorization = `Basic ${Buffer.from(`${userId}:${apiKey}`).toString('base64')}`;
+
+async function send(path, method, payload) {
+  const response = await fetch(`${endpoint}${path}`, {
+    method,
+    headers: { authorization, 'content-type': 'application/json' },
+    body: payload,
+  });
+  if (!response.ok) {
+    console.error(`${method} ${path} failed (${response.status}):`);
+    console.error(await response.text());
+    process.exit(1);
```

ESLint

[error] Don't use process.exit(); throw an error instead.

(n/no-process-exit)